arm-sql/compositeSql.json

```diff
@@ -13,8 +13,6 @@
     "./2014-04-01/swagger/servers.json",
     "./2014-04-01/swagger/sql.core.json",
     "./2014-04-01/swagger/databaseSecurityAlertPolicies.json",
-    "./2015-05-01-preview/swagger/blobAuditingPolicies.json",
```

Comment:
Is that a blocker?  It's a reasonable thing to do, but I didn't check in these files, and they both already went through the Pull request review process, so I'd hate for anyone to have to review already-reviewed files.  

If there's a policy against having swagger files not referenced by the composite file, we may have an issue later on, as we have two versions of some APIs and I will be adding the new version and replacing the reference to the old version in the composite file, but not deleting the old file.